components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1614,13 +1620,13 @@ public void revokeAccessTokensIndividually(String[] tokens, boolean isHashedToke
                 }
                 accessTokenId.add(getTokenIdByAccessToken(token));
             }
-            // To revoke request objects which have persisted against the access token.
-            if (accessTokenId.size() > 0) {
-                OAuth2TokenUtil.postUpdateAccessTokens(accessTokenId, OAuthConstants.TokenStates.
-                        TOKEN_STATE_REVOKED);
-            }
 
             if (isTokenCleanupFeatureEnabled) {
+                if (connection.getMetaData().getDriverName().contains("Microsoft")) {
```

Comment:
Even in the token clean up external stored procedures, only for MSSQL server setups the IDN_OIDC_REQ_OBJECT_REFERENCE table entries are deleted as for other database types on delete cascade happens.

[1] - https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/mssql/token-cleanup/mssql-tokencleanup.sql#L237